.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
+
+
+mkdir target/classes
+mkdir target/eclipse
```

Comment:
such folders might already exist on local filesystem, please use "-p" argument to avoid failure.